concepts/Common/Integration Guides/30 Create a DevExtreme application with Vite.js/30 Create a DevExtreme application with Vite.js.md

```diff
@@ -0,0 +1,147 @@
+Vite (pronounced "veet") is a web development framework from the creators of Vue.js. It offers an intuitive API, fast build times, and efficient dependency bundling thanks to its use of native ESM modules.
+
+Unlike its major competitors — Next.js and `create-react-app` — Vite is compatible with a variety of front-end solutions. In addition to its original companion framework — Vue.js — you can use Vite with React.js, Svelte, or even vanilla JavaScript.
+
+In this tutorial, we'll create a new Vite application with a Vue.js frontend. The application will pull data from the publicly available [Cat Facts API](https://catfact.ninja) and display it with the DevExpress Data Grid.
+
+## 1. Create a new Vite project
+
+Use the following command to launch the Vite project wizard:
+
+    npm create vite@latest
+
+![Vite CLI wizard](/images/Vite/create-vite.png)
+
+Answer the wizard's prompts to select your project name, your front-end framework, and your programming language (JavaScript or TypeScript).
+
+The Vite wizard does not install project dependencies automatically. Enter your project directory and run `npm install`:
+
+    cd vite-project; npm install
+
+To start the development server, execute the following command:
+
+    npm run dev
+
+![Vite server up and running](/images/Vite/vite-ready.png)
+
+## 2. Configure DevExtreme dependencies
+
+Add the following dependencies to the `package.json` file:
+
+    <!--tab: package.json-->
+    "dependencies": {
+        "devextreme": "^23.2",
+        "devextreme-vue": "^23.2",
+        ...
+    }
+
+Install the new dependencies:
+
+    npm install
+
+## 3. Remove unnecessary code
+
+1. Open the `srtc/App.vue` file, and replace its content with the following two tags:
+
+        <!--tab: App.vue-->
+        <script setup lang="ts">
+        </script>
+
+        <template>
+        </template>
+
+2. Remove the `src/components/HelloWorld.vue` file because we no longer need it.
+
+3. The `src/style.css` file includes CSS code that overrides DevExtreme styles. Open the file and remove its contents.
+
+## 4. Set up a DevExtreme theme
+
+1. Add the following `import` statement to the `src/main.ts` file:
+
+        <!--tab: main.ts-->
+        import 'devextreme/dist/css/dx.fluent.blue.light.css';
+
+    This statement applies the `fluent` theme to your application. You can select a different DevExtreme theme if you wish.
+
+2. To apply this theme throughout your application, edit the `index.html` file in the project's root directory. Add the `dx-viewport` class to the `body` tag:
+
+        <!--tab: index.html-->
+        <body class="dx-viewport">
+
+## 5. Add a new component
+
+1. Create a new file for the component --- `src/components/CatFactGrid.vue`. Populate it with the following two tags:
+
```

Comment:
We usually declare templates first, then scripts in our demos and docs

Reply:
@titerman please don't forget to upgrade version to 24.1